mlflow/metrics/base.py

```diff
@@ -78,26 +78,36 @@ class EvaluationExample:
     justification: str = None
     grading_context: Dict[str, str] = None
 
+    def _format_grading_context(self):
+        return "\n".join(
+            [f"key: {key}\nvalue: {value}" for key, value in self.grading_context.items()]
+        )
+
     def __str__(self) -> str:
         grading_context = (
             ""
             if self.grading_context is None
-            else "\n".join(
-                [f"Provided {key}: {value}" for key, value in self.grading_context.items()]
-            )
+            else f"======================\nAdditional information\n======================\n"
```

Comment:
Maybe better to use "Additional information\n\n"? This can save some tokens and also LLM prefers new line over =======